src/main/kotlin/uk/gov/justice/digital/hmpps/digitalprisonreportingmi/controller/ExternalMovementsController.kt

```diff
@@ -19,4 +25,22 @@ class ExternalMovementsController() {
   fun stubbedCount(): Count {
     return Count(501)
   }
+
+  @GetMapping("/external-movements")
+  @Operation(
+    description = "Gets a count of external movements (mocked)",
```

Comment:
I think think should be "list" rather than "count".

Reply:
Good spot, I have changed that.